multiview/embed/splitae.py

```diff
@@ -0,0 +1,226 @@
+import torch
+from torch.utils.data import Dataset, DataLoader
+from torchvision import datasets
+import torchvision
+import matplotlib.pyplot as plt
+import PIL
+import numpy as np
+import itertools
+import tqdm
+
+from .base import BaseEmbed
+from ..utils.utils import check_Xs
+
+
+class FullyConnectedNet(torch.nn.Module):
+    def __init__(self, inputSize, hiddenSize, numHiddenLayers, embeddingSize):
```

Comment:
Seems like this class and its functions should have docstrings

Reply:
It's an internal class so I guess it just needs an underscore

Reply:
No, provide documentation still so people who are reading or using your code will know what's happening.